processor/elasticinframetricsprocessor/README.md

```diff
@@ -3,13 +3,11 @@
 <!-- status autogenerated section -->
 | Status        |           |
 | ------------- |-----------|
-| Stability     | [alpha]:  metrics  |
-| Distributions |  |
-| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@ishleenk17](https://www.github.com/ishleenk17) |
+| Stability     | [alpha]: metrics   |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aprocessor%2Felasticinframetrics%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aprocessor%2Felasticinframetrics) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aprocessor%2Felasticinframetrics%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aprocessor%2Felasticinframetrics) |
```

Comment:
I am not sure why this is getting added by autogen code. I will remove this.

Reply:
I think it's part of the `mdatagen` templates, should we  keep it to prevent the linter to fail?

Reply:
I see that the CI does not complain, good to go :+1:

Reply:
I ran it on local and no failures in lint. It doesn't look like we check that mdatagen diverges. However, added it back to keep in sync with the autogen stuff. We can deal with this later.